Description: Disable unavailable tests
 Test material is unavailable from crates.io archive
Last-Update: 2022-10-24
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/tests.rs
+++ b/src/tests.rs
@@ -32,7 +32,7 @@
     assert_eq!(create_test_input(&get_lines(lines)), expected);
 }
 
-#[test]
+//#[test]
 fn test_markdown_files_of_directory() {
     let files = vec![
         "../testing/tests/hashtag-test.md",
